README.md

```
@@ -12,29 +12,148 @@ yarn add dag-rs
 ```
 
 ## Usage
+
+### CommonJs
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)